src/components/KeyboardShortcuts/index.spec.tsx

```diff
@@ -167,6 +228,205 @@ describe(__filename, () => {
     },
   );
 
+  it.each([
+    ['previous', 'p', RelativePathPosition.previous],
+    ['next', 'n', RelativePathPosition.next],
+  ])(
+    'dispatches goToRelativeDiff with %s when "%s" is pressed',
+    (direction, key, position) => {
+      const addonId = 1;
+      const baseVersionId = 2;
+      const currentPath = fakeVersionWithDiff.file.selected_file;
+      const hash = '#D1';
+      const headVersionId = 3;
+      const pathList = [currentPath];
+      const versionId = headVersionId;
+      const version = {
+        ...fakeVersionWithDiff,
+        id: headVersionId,
+        file: {
+          ...fakeVersionWithDiff.file,
+          // eslint-disable-next-line @typescript-eslint/camelcase
+          selected_file: currentPath,
+        },
+      };
+      const history = createFakeHistory({
+        location: createFakeLocation({
+          search: queryString.stringify({ path: currentPath }),
+          hash,
+        }),
+      });
+
+      const store = configureStoreWithFileTree({ versionId, pathList });
+      store.dispatch(versionsActions.loadVersionInfo({ version }));
```

Comment:
This seems like something you'd want to have happen automatically. There is a new helper you could use:

```diff
diff --git a/src/components/KeyboardShortcuts/index.spec.tsx b/src/components/KeyboardShortcuts/index.spec.tsx
index 8e1dcb7..d99223f 100644
--- a/src/components/KeyboardShortcuts/index.spec.tsx
+++ b/src/components/KeyboardShortcuts/index.spec.tsx
@@ -21,6 +21,7 @@ import {
   createFakeHistory,
   createFakeLocation,
   createFakeThunk,
+  createStoreWithVersion,
   fakeVersion,
   fakeVersionEntry,
   fakeVersionWithDiff,
@@ -80,28 +81,34 @@ describe(__filename, () => {
 
   const configureStoreWithFileTree = ({
     pathList = ['file1.js'],
-    store = configureStore(),
     versionId = 321,
+    externalVersion = {
+      ...fakeVersion,
+      id: versionId,
+      file: {
+        ...fakeVersion.file,
+        entries: pathList.reduce((pathMap, path: string) => {
+          return {
+            ...pathMap,
+            [path]: {
+              ...fakeVersionEntry,
+              filename: path,
+              path,
+            },
+          };
+        }, {}),
+      },
+    },
+    store = createStoreWithVersion(externalVersion),
+  }: {
+    pathList?: string[];
+    versionId?: number;
+    externalVersion?: typeof fakeVersion;
+    store?: Store;
   } = {}) => {
     store.dispatch(
       fileTreeActions.buildTree({
-        version: createInternalVersion({
-          ...fakeVersion,
-          id: versionId,
-          file: {
-            ...fakeVersion.file,
-            entries: pathList.reduce((pathMap, path: string) => {
-              return {
-                ...pathMap,
-                [path]: {
-                  ...fakeVersionEntry,
-                  filename: path,
-                  path,
-                },
-              };
-            }, {}),
-          },
-        }),
+        version: createInternalVersion(externalVersion),
       }),
     );
 
```

Reply:
Nice, thanks!